data/136/075/304/1/1360753041.geojson

```diff
@@ -15,6 +15,9 @@
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
     "name:eng_x_preferred":[
+        "St. Georges and Priorslee"
+    ],
+    "name:eng_x_variant":[
         "St. Georges And Priorslee"
```

Comment:
Please add St. spelled out as variant

Reply:
Adding this in a new commit, along with some changes to label properties. There are inconsistencies I've noticed in label and name values, I'll clean them up in a new commit.